### ISSUE TITLE
[enhancement] [UI]: Add ability to sort/autosort models in ModelUI

### ISSUE THREAD

**Issue by Malrama** [invoke-ai/InvokeAI#5885]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What should this feature add?

Add the ability to sort the Model View and auto-sort once a new model is being installed.

Currently, new installed models are just added to the end of a list without the ability to sort them.

The following sorting options seem to be most useful to me:
°Name
°Type
°Install date (is that possible?)
Maybe some others are useful too. Being able to mix sorting options would be great, but probably a bit complicated.

This would make model management and model search much more convenient.

Greetings

**Comment by lstein** [comment]
Agree. This is on my wishlist too. I'm mostly concerned about the model pulldowns in the loader node and linear views, where I'd prefer to see the models sorted first by base type (as they are) and then alphabetically by name using case-intensive sorting.

**Comment by skunkworxdark** [comment]
Was about to raise a feature request for exactly this but found this one first.

Additionally, I would like to see an option to sort by size, path, base type and last updated date.  

The date created would be useful for seeing a model you have just added, as sometimes they are not named what you would expect.

The size would be useful for sifting out larger models that you no longer use.

**Comment by skunkworxdark** [comment]
I knocked up a PR for this.  https://github.com/invoke-ai/InvokeAI/pull/9024

**Comment by michaelanderson01826-glitch** [comment]
Thanks for submitting the PR for this enhancement. I appreciate you taking
the time to put that together; the ability to sort by size and date will be
a great addition to the ModelUI.
